pages/app/views/refinery/admin/pages/_page.html.erb

```diff
@@ -25,7 +25,7 @@
 
     <% if Refinery::I18n.frontend_locales.many? %>
       <span class='locales'>
-        <% page.translations.sort_by{ |t| Refinery::I18n.frontend_locales.index(t.locale)}.each do |translation| %>
+        <% page.translations.sort_by{ |t| Refinery::I18n.locales.index(t.locale)}.each do |translation| %>
```

Comment:
I'm not sure if we want this, perhaps it's a good idea to ask a dev if we can add/remove a frontend language.